common/api/core-frontend.api.md

```diff
@@ -9118,19 +9142,9 @@ export class SheetViewState extends ViewState2d {
 // @internal
 export type ShouldAbortReadGltf = (reader: GltfReader) => boolean;
 
-// @public
-export abstract class SkyBox implements SkyBoxProps {
-    protected constructor(sky?: SkyBoxProps);
-    static createFromJSON(json?: SkyBoxProps): SkyBox;
-    display: boolean;
-    // @internal (undocumented)
-    abstract loadParams(_system: RenderSystem, _iModel: IModelConnection): SkyBoxParams;
-    // (undocumented)
-    toJSON(): SkyBoxProps;
-}
-
-// @public
+// @internal
 export namespace SkyBox {
+    // @public

```

Comment:
Why are these public members when the namespace itself is internal? I don't think they'd show in the docs this way.

Reply:
They won't show in the docs, but they should not be marked public.
I left a ###TODO to clean this bit up, will address now.

Reply:
Addressed.